executor/runtime.pxd

```diff
@@ -4,6 +4,7 @@ cdef extern from "../runtime_util/runtime_util.h":
     int MAX_DEVICES
     int MAX_PARAMS
     int NUM_GAMEPAD_BUTTONS
+    int NUM_KEYBOARD_BUTTONS
```

Comment:
Yes it needs to be updated. Its pretty simple and you can't really forget cus otherwise you can't use runtime in Cython lol. I don't think there's anyway to automate this.